### PR TITLE
Analytics filter according to project

### PR DESCRIPTION
**Pull request by sainivikas**
1. Analytics filter according to project.
2. title added in partner list.
3. Delhi Office address changed.

### CODE REVIEW

File: programs/models.py [hunk 1]
```diff
@@ -6,7 +6,20 @@ class Partner(CocoModel):
...
     def __unicode__(self):
         return self.partner_name
+
+
+class Project(CocoModel):
+    id = models.AutoField(primary_key=True)
+    project_name = models.CharField(max_length=100, unique=True)
+    start_date = models.DateField(null=True, blank=True)
+    end_date = models.DateField(null=True, blank=True)
+    associate_partner = models.ManyToManyField(Partner, blank=True)
```
Comment (Abhishek-lodha) [inline programs/models.py]: We can add a description/full form field for projects

File: output/views/common.py [hunk 2]
```diff
@@ -175,6 +177,21 @@ def get_geog_id(request):
...
+#Return a dictionary of list of PROJECT_NAME,ID AND THEIR ASSOCIATE PARTNER'S ID and 
+#second dictionary of project wise partners
+def get_project_list(projects):
+    coco_projects = Project.objects.values('id', 'associate_partner')
+    project_wise_partner = dict()
+    for project in coco_projects:
+        if str(project['id']) not in project_wise_partner:
+            project_wise_partner[str(project['id'])] = list()
+        project_wise_partner[str(project['id'])].append(str(project['associate_partner']))
+    coco_projects = Project.objects.values('id', 'project_name').order_by('project_name')
```
Comment (Abhishek-lodha) [inline output/views/common.py]: we can use a common queryset for both.

File: output/views/common.py [hunk 3]
```diff
@@ -175,6 +177,21 @@ def get_geog_id(request):
...
+    if projects:
+        for project in coco_projects:
```
Comment (Abhishek-lodha) [inline output/views/common.py]: can we merge both the for loops

File: output/views/overview_analytics.py [hunk 4]
```diff
@@ -1,8 +1,10 @@
 from django.shortcuts import *
 from django.http import Http404, HttpResponse
 import datetime
+import json
 
 from geographies.models import Block, Country, District, State, Village
+from programs.models import Partner, Project
```
Comment (sainivikas) [inline output/views/overview_analytics.py]: removed.

File: programs/models.py (hunk 1 above)
Comment (sainivikas) [inline programs/models.py]: Nice thought, added.

File: output/views/common.py (hunk 2 above)
Comment (sainivikas) [inline output/views/common.py]: done.

File: output/views/common.py (hunk 3 above)
Comment (sainivikas) [inline output/views/common.py]: definitely. merged.

**Pull request merged**